File: LeetCode/0136_Single_Number/solution.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var singleNumber = function(nums) {
    for(let i=0;i<nums.length;i++){
        let a =true;
        for(let j=0; j<nums.length;j++){
            if(i==j) continue;
            if(nums[i]==nums[j])
                a = false;
        }
        if(a) return nums[i]
    }
};

File: LeetCode/0136_Single_Number/solution.cpp
class Solution {
public:
    int singleNumber(vector<int>& nums) {
        unordered_set<int> sets;
        for(int i = 0; i<nums.size();i++){
            if(sets.count(nums[i])) sets.erase(nums[i]);
            else sets.insert(nums[i]);
        }
        return *sets.begin();
    }
};